content/docs-community/concepts/contributing-to-docs/contributing-to-docs.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Contributing to Docs'
+Description: 'Contributing to Docs allows community members to reinforce their knowlede and hone their skills with Git and Markdown.'
+Subjects:
+  - 'Code Foundations'
+Tags:
+  - 'Documentation'
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'introduction-to-open-source'
+  - 'paths/code-foundations'
+---
+
+**Contributing to Docs** allows community members of all skill sets to not only reinforce what they've (perhaps already) learned, but also to hone their skills in version control with [Git](https://www.codecademy.com/resources/docs/git), peer-to-peer colloboration on [GitHub](https://www.codecademy.com/resources/docs/general/github), and writing content in [Markdown](https://www.codecademy.com/resources/docs/markdown). Contributors get to work with the content team that maintains Docs in a review process of their contributions.
```

Comment:
That last sentence reads odd to me: "in a review process of their contribution." Maybe something like "as their contribution goes through the review process."